documentation/smg/14_release_notes.md

```diff
@@ -0,0 +1,57 @@
+# Writing Release Notes
+
+## Release Notes Files
+
+When adding a feature, enhancement or bug fix,
+a developer will update the upcoming release's release notes.
+The release notes documents are held in `documentation/release_notes`.
+There is a release notes file for every major, minor and patch release.
+The documents are ReST files and
+have the same name as the corresponding release, e.g. `v3.5.0.rst`.
+
+## Recording Release Notes
+
+On every pull request, the developer must consider whether the changes require
+documenting.
+If the change will affect users in any way, be it a bug fix, a new feature or
+an API change, then a sentence or two must be added to the latest release notes
```

Comment:
With version-named files it is not simple to define the 'latest release"' - the release version may not be decided until closer to release time. 

It would be better simpler have a `next-release` file to collect the developer notes from which the final RN is generated once the release version number is agreed. 

When that RN is created, all points from the file relating to the new release can be removed.

Reply:
I guess the decision is between:

1) Having a `next-release.md` file:
    - a new release notes file is created just before the release with the correct release number
    - everything will be migrated across from `next-release` and `next-release` will be wiped

2) Assuming a patch increment at first
    - new release notes file is created on first PR into that release, this will be `v.3.5.1` if the previous release was `v3.5.0`.
    - the name of that file is changed when it comes time to decide what the release will actually be called.

I went for 2 initially, this has the benefit of all updates for each release being in a single file, with a single history (which may contain a rename). But I can see 1 has its merits too.